godtools/App/Features/AppLanguage/Domain/UseCases/SearchAppLanguageInAppLanguagesListUseCase.swift

```diff
@@ -0,0 +1,48 @@
+//
+//  SearchAppLanguageInAppLanguagesListUseCase.swift
+//  godtools
+//
+//  Created by Rachael Skeath on 10/4/23.
+//  Copyright © 2023 Cru. All rights reserved.
+//
+
+import Foundation
+import Combine
+
+class SearchAppLanguageInAppLanguagesListUseCase {
+    
+    private let getAppLanguagesListUseCase: GetAppLanguagesListUseCase
+    
+    init(getAppLanguagesListUseCase: GetAppLanguagesListUseCase) {
+        self.getAppLanguagesListUseCase = getAppLanguagesListUseCase
+    }
+    
+    func getSearchResultsPublisher(for searchTextPublisher: CurrentValueSubject<String, Never>) -> AnyPublisher<[AppLanguageListItemDomainModel], Never> {
+        
+        return Publishers.CombineLatest(
```

Comment:
Hey @rachaelblue this is perfect, this is exactly what we're looking for and the basically what Combine was meant to do.